dev-support/code-formatter/hadoop_idea_formatter.xml

```diff
@@ -0,0 +1,76 @@
+<!--
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-->
+<code_scheme name="Hadoop" version="173">
+  <option name="RIGHT_MARGIN" value="80" />
```

Comment:
It seems duplicated.

Reply:
I also thought that first but since the entire template is generated by IntelliJ only, I searched for this difference and found that this one is top/global level tag applicable to all: Java, XML, Python, Html etc, and the one within `codeStyleSettings language="JAVA"` is for Java only.

Based on the definition provided [here](https://www.jetbrains.com/help/idea/code-style-jspx.html#7af2e39d), 
Right Margin: Use these settings to specify a margin space required on the right side of an element. If you select Default option then a value of the right margin from the [global settings](https://www.jetbrains.com/help/idea/settings-code-style.html#rightMargin) will be used.